controllers/usersignup/usersignup_controller_test.go

```diff
@@ -1598,7 +1597,6 @@ func TestUserSignupMUROrSpaceOrSpaceBindingCreateFails(t *testing.T) {
 			assert.Equal(t, "approved", userSignup.Labels[toolchainv1alpha1.UserSignupStateLabelKey])
 			AssertMetricsCounterEquals(t, 1, metrics.UserSignupApprovedTotal)
 			AssertMetricsCounterEquals(t, 1, metrics.UserSignupUniqueTotal)
-			segmenttest.AssertMessageQueued(t, r.SegmentClient, userSignup, segment.AccountActivated)
 		})
```

Comment:
also here
```suggestion
			segmenttest.AssertNoMessageQueued(t, r.SegmentClient)
		})
```
and in all other cases below